Pipfile

```diff
@@ -37,6 +37,7 @@ packaging = "*"
 cloudsplaining = ">=0.4.1"
 networkx = "*"
 dockerfile-parse ="*"
+dataclasses = {version = "*", markers="python_version < '3.7'"}
```

Comment:
@metahertz You'll need to update `Pipfile` and `Pipfile.lock` too

Reply:
Thanks!

Issue i'm having is there doesnt seem to be a way to specify a range with `>=` in `[requires]`
ie:
```
python_version = ">=3.6"
```

Still gives warnings you have a 3.7 pipenv, which isn't very neat.